graylog2-web-interface/src/views/pages/ShowViewPage.jsx

```diff
@@ -82,6 +84,8 @@ class ShowViewPage extends React.Component<Props, State> {
     ).then((results) => {
       this.setState({ loaded: true });
       return results;
+    }).then(() => {
```

Comment:
```suggestion
    }).then(() => SearchActions.executeWithCurrentState())
```

This is not only syntactic sugar, it also leads to the result of `executeWithCurrentState` (a promise) becoming part of the promise chain, so the following `catch` is also applied to it.